plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1759,7 +1760,7 @@ protected OptionalInt maxTableNameLength()
     @Override
     protected void verifyTableNameLengthFailurePermissible(Throwable e)
     {
-        assertThat(e).hasMessageMatching("(?s)(.*Read timed out)|(.*\"`TBL_NAME`\" that has maximum length of 128.*)");
+        assertThat(e).hasMessageMatching("Table name must be shorter than or equal to '128' characters but got.*");
```

Comment:
it looks like we're losing test coverage for proper handling of too long names, but in fact we didn't have it before, as "read timed out" was an allowed outcome. thus it's not a big loss.

we should, however, test name limits on the smoke test, since that's what is run against different metastores/catalogs in delta & iceberg.